### PR TITLE
Add functionality from insert-variable-plugin

### PR DESCRIPTION
**Pull request by elpoelma**
This PR moves the functionality from the insert-variable-plugin to this addon. Both the insert-variable-plugin and template-variable-plugin are tightly coupled.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +25,41 @@ ember install ember-rdfa-editor-template-variable-plugin
...
+},
+```
+
+When using the insert-variable-plugin, you can also filter the codelists by publisher. You can pass the publisher uuid when initializing the plugin. Additionally you can also pass an array to the plugin containing the variable types you want to support.
+
+```javascript
+{
+  name:'insert-variable',
+  options: {
+    publisher: 'cec59e5e872a9084e93becf3026bfcc2f25926ea76372711b7a745875f3b7949',
```
Comment (lagartoverde) [inline README.md:51]: I think the publisher is now an uri? I have to double check this

File: tests/dummy/app/formats.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+export default {
```
Comment (lagartoverde) [inline tests/dummy/app/formats.js:1]: What is this file for?

**Review by lagartoverde — APPROVED** [review]
Seems to work, some comments but overall good work :)

File: README.md (hunk 1 above)
Comment (lagartoverde) [reply README.md:51]:
```suggestion
    publisher: 'http://data.lblod.info/id/bestuurseenheden/141d9d6b-54af-4d17-b313-8d1c30bc3f5b',
```

File: tests/dummy/app/formats.js (hunk 2 above)
Comment (elpoelma) [reply tests/dummy/app/formats.js:1]: Internationalization

**Review by nvdk — APPROVED** [review]

**Comment by lagartoverde** [comment]
I added https://binnenland.atlassian.net/browse/GN-3367 on here as it doesn't change anything in the plugin, and didn't want to add on the pre-merged version of the plugins

**Pull request merged**